### PR TITLE
Update debugger version

### PR DESCRIPTION
**Pull request by Yuyupo**
IoT.jsCode-DCO-1.0-Signed-off-by: Daniella Barsony bella@inf.u-szeged.hu

### CODE REVIEW

File: src/app/modules/client/connection.js [hunk 1]
```diff
@@ -86,14 +86,14 @@ export default class Connection {
...
         }
 
-        this._debuggerObj.maxMessageSize = message[1];
-        this._debuggerObj.cPointerSize = message[2];
-        this._debuggerObj.littleEndian = (message[3] != 0);
-        this._debuggerObj.version = message[4];
+        this._debuggerObj.maxMessageSize = message[6];
+        this._debuggerObj.cPointerSize = message[7];
+        this._debuggerObj.littleEndian = (message[1] != 0);
+        this._debuggerObj.version = message[2];
```
Comment (robertsipka) [inline src/app/modules/client/connection.js]: This is not correct here. You should returns with a 32-bit integer read from the message array at offset (at 2) to offset + 3, in either direction (which is defined by the endianess). 
BTW, it is used somewhere? We should check that the `JERRY_DEBUGGER_VERSION` is equal with the received version number.

Comment (robertsipka) [reply src/app/modules/client/connection.js]: Something like this (you should create a function for this if it is not exist yet) :
```
let value = 0;
let offset = 2;
if (this._debuggerObj.littleEndian) {
    value = message[offset];
    value |= message[offset + 1] << 8;
    value |= message[offset + 2] << 16;
    value |= message[offset + 3] << 24;
} else {
    value = message[offset] << 24;
    value |= message[offset + 1] << 16;
    value |= message[offset + 2] << 8;
    value |= message[offset + 3];
}
```

**Comment by Yuyupo** [comment]
@robertsipka Added the missing/new packages that was requested and modified the rest according to what you mentioned.

**Comment by LaszloLango** [comment]
#137 is landed. I think the commit message should be updated before the merge.

**Comment by Yuyupo** [comment]
@LaszloLango Updated it :)

**Comment by Yuyupo** [comment]
@LaszloLango  Alright. :) Added a description to the commit message. Does it suffice?

**Comment by LaszloLango** [comment]
@Yuyupo perfect, thanks. PR is merged.

**Pull request merged**